src/Component/page/Reservations/Form.js

```diff
@@ -8,23 +8,24 @@ const Form = () => {
   return (
     <div className='Form-grid-container'>
       <div className='logo-form'>
-       <img src={Formp}/>
+       <img src={Formp} alt= "logoformp"/>
       </div>
     <div className='reservation-form-container'>
       <div className='mb-1'>
-        <h1>Reserve a table</h1>
+        <h1>Book a table</h1>
           <p>We take bookings for groups to 6. We get busy on weekends so best to book early to avoid disappointment!</p>
       </div>
       {/* First Row */}
 
       <div className='d-grid three-col-grid my-1 grid-gap-1 text-sage '>
       <div>
         <label className='d-block text-sage '>Date</label>
-        <input type="text text-sage" className='form-input'/>
+        <input type="date text-sage" className='form-input'  />
```

Comment:
date text-sage is not a valid HTML input type so this displays as a plain text input